platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/paos/PaosOutInterceptor.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.cxf.paos;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.core.HttpHeaders;
+
+import org.apache.cxf.interceptor.Fault;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.phase.AbstractPhaseInterceptor;
+
+public class PaosOutInterceptor extends AbstractPhaseInterceptor<Message> {
+
+    public PaosOutInterceptor(String phase) {
+        super(phase);
+    }
+
+    @Override
+    public void handleMessage(Message message) throws Fault {
+        Map<String, List<String>> headers = (Map<String, List<String>>) message.get(
+                Message.PROTOCOL_HEADERS);
+        List<String> acceptHeaders = headers.get(HttpHeaders.ACCEPT);
+        if (acceptHeaders == null) {
+            acceptHeaders = new ArrayList<>();
+        }
+
+        acceptHeaders.add("application/vnd.paos+xml");
```

Comment:
Should we add some kind of way of disabling ECP/PAOS support in the secure rest client?  Might be helpful if we run into interoperability issues with other ECP implementations due to vagueness or misunderstandings in the spec.

If we do the client then we should also consider if we can disable the ECP server as well.

It would be nice if we could setup an another ECP server and client implementation to test compatibility.  Being able to disable the feature might be a good fallback if it takes too long to setups other ECP implementations.